### PR TITLE
Added VB vstemplates for three SideWaffle item templates

### PR DESCRIPTION
**Pull request by RandomlyKnighted**

### CODE REVIEW

**Comment by RandomlyKnighted** [comment]
After this is merged can we do a release in order to allow anyone who read the SideWaffle article in MSDN Magazine the ability to add these templates in their VB projects?

File: TemplatePack/template-report.xml [hunk 1]
```diff
@@ -91,11 +91,7 @@
     <ItemTemplate Path="ItemTemplates\Azure\BlobUploadHelper\_Definitions\CSharp.vstemplate" Name="Windows Azure Blob Upload Helper" Description="Adds a class file to any existing VSIX project that provides convenience utilities for upload files to Windows Azure Blob Storage." ProjectType="CSharp" ProjectSubType="" />
     <ItemTemplate Path="ItemTemplates\Azure\CloudServicePlugin\_Definitions\CSharp.vstemplate" Name="Windows Azure Cloud Service Plugin" Description="A template to create a plugin which extends the Virtual Studio SDK for Cloud Services." ProjectType="CSharp" ProjectSubType="" />
     <ItemTemplate Path="ItemTemplates\Azure\TableStorageEntity\_Definitions\CSharp.vstemplate" Name="Windows Azure Table Storage Entity" Description="Adds a class file to any existing VSIX project that provides entity representation of data stored in Windows Azure Storage Tables." ProjectType="CSharp" ProjectSubType="" />
-<<<<<<< HEAD
```
Comment (sayedihashimi) [inline TemplatePack/template-report.xml]: There is some invalid XML here from a merge, can you clean it up by removing the ```<<<<<``` and other invalid lines?

**Comment by sayedihashimi** [comment]
Thanks for the PR. I've added a few comments.

File: TemplatePack/template-report.xml (hunk 1 above)
Comment (RandomlyKnighted) [inline TemplatePack/template-report.xml]: I just looked and that isn't in there anymore. What you're seeing in the PR is GitHub showing you that I removed those lines in one of my commits.

**Comment by sayedihashimi** [comment]
Thanks, merged it.

**Pull request merged**